masterlist.yaml

```
@@ -18868,6 +18868,30 @@ plugins:
         util: '[SSEEdit v4.0.3](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 14
 
+  - name: 'Ryn''s Farms.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/72305/'
+        name: 'Ryn''s Farms'
+    msg:
+      - <<: *patch3rdParty
+        subs:
+          - 'Skyrim Sewers'
+          - '[Patches for Ryn''s](https://www.nexusmods.com/skyrimspecialedition/mods/74233/)'
+        condition: 'active("Ryn''s Farms.esp") and not active("Ryn''s Farms - Skyrim Sewers.esp")'
+      - <<: *patch3rdParty
+        subs:
+          - 'Rift Watchtower NavFix'
```

I'd still use the mod's name & then specify the package (or plugin if necessary) that the patch is for:

```suggestion
          - 'Navigator - Navmesh Fixes (Rift Watchtower NavFix)'
```